robot/impl/resource_manager.go

```diff
@@ -182,9 +182,31 @@ func (manager *resourceManager) updateRemoteResourceNames(
 	rr internalRemoteRobot,
 	recreateAllClients bool,
 ) bool {
-	manager.logger.CDebugw(ctx, "updating remote resource names", "remote", remoteName, "recreateAllClients", recreateAllClients)
+	logger := manager.logger.WithFields("remote", remoteName)
```

Comment:
flyby: we have a lot of logs in this function that all include the remote name as metadata - this local logger formalized this